src/main/java/com/exasol/projectkeeper/validators/changesfile/ChangesFileValidator.java

```diff
@@ -20,20 +21,24 @@
     private final Path changesFileAbsolutePath;
     private final Path projectDirectory;
     private final String projectVersion;
+    private final MavenModelReader mavenModelReader;
 
     /**
      * Create a new instance of {@link ChangesFileValidator}
      * 
      * @param projectVersion   version of the project to validate
      * @param projectName      name of the maven project
      * @param projectDirectory root directory of the maven project
+     * @param mavenModelReader reader for maven models
      */
-    public ChangesFileValidator(final String projectVersion, final String projectName, final Path projectDirectory) {
+    public ChangesFileValidator(final String projectVersion, final String projectName, final Path projectDirectory,
```

Comment:
Just an ides for future: make an object wrapper for project info: version, name, path